### DISCUSSION TITLE
How to combine regexManagers with groupName

### DISCUSSION THREAD

**Discussion by hubertgrzeskowiak** [renovatebot/renovate#26218] [Request Help]
### How are you running Renovate?

None

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.71.2

### If you're self-hosting Renovate, select which platform you are using.

k8s

### What is your question?

[`regexManagers`][1] allow matching dependencies through RegEx. We have a use case where we would like to match the same dependency in multiple files. Normally we'd use [`groupName`][2] for that, but it seems that that field is not compatible with RegEx:

>  validationMessage: Custom Manager contains disallowed fields: groupName

Is there an alternative?

[1]: https://docs.renovatebot.com/modules/manager/regex/
[2]: https://docs.renovatebot.com/configuration-options/#groupname

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Use packageRules in combination with regexManagers
